optimum/exporters/openvino/convert.py

```diff
@@ -722,6 +724,15 @@ def export_tokenizer(
     if output.exists():
         tokenizer = maybe_convert_tokenizer_to_fast(tokenizer, output)
 
+    if (
+        task is not None
+        and task.startswith("text-generation")
```

Comment:
Is this only related to exact "text-generation" as there are few variants of it? how about `text2text-generation`?

Reply:
quoting a response from a transfromers maintainer about left padding for text2text-generation: 

> It can be in the following corner case (all of these must apply):
> - you are doing batch decoding
> - you are passing the start of the decoding ids, in addition to the inputs to the encoder
> - the passed decoding ids don’t have the same length, they must be left-padded
> 
> (if the encoder inputs don’t have the same length, they must be right-padded :eyes: )

> By default no decoder inputs are passed, and thus no left padding is ever needed :slightly_smiling_face:

so overall it seems never needed.